### DISCUSSION TITLE
Is there a way to hide the last update date on single pages

### DISCUSSION THREAD

**Discussion by sgustafsson** [squidfunk/mkdocs-material#6148] [Q&A] [answered]
Hi

I  am using the [Revisioning](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/?h=date#revisioning) feature to display the date of last update per page. Is there a way to hide this last update information on single pages (for example index.md), just like other things can be hidden?

`hide: - navigation -  creation_date` 

cheers

**Comment by lampensau** [comment, marked as answer]
As the linked documentation of mkdocs-material includes various features provided by supported _external_ plugins, it would be helpful to know what part you want to disable.

- The date? Have a look at https://timvink.github.io/mkdocs-git-revision-date-localized-plugin/options/#exclude
- The contributors? Have a look at https://github.com/ojacques/mkdocs-git-committers-plugin-2#config
- The authors? Have a look at https://timvink.github.io/mkdocs-git-authors-plugin/options.html#exclude

As stated earlier, these are all functions provided by external plugins that receive support (styling, etc.) from mkdocs-material. The provided documentation only covers a basic setup.

For extended documentation, you should check out the respective plugin documentation as linked above.

**Reply by sgustafsson** [reply]
Thanks! https://timvink.github.io/mkdocs-git-revision-date-localized-plugin/options/#exclude did it for me.
